app/views/layouts/application.html+govwales.erb

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
We were hoping to get away without having to change this main layout file.  If we have a variant, it's going to make it difficult to retrofit any changes that GDS make on there side.

Can you give us more information about what's changed in this variant, @markheathy ?

Reply:
The breadcrumb position has aboved above the title block @rickclare to fit WG UX.

Reply:
Its a lift and shift, not a radical structural change - Element Ordering.